clouddq/templates/dbt/macros/create_entity_aggregate_dq_summary.sql

```diff
@@ -12,12 +12,13 @@
 -- See the License for the specific language governing permissions and
 -- limitations under the License.
 
-{% macro create_entity_aggregate_dq_summary(entity_target_rule_binding_configs, gcp_project_id, gcp_bq_dataset_id) -%}
+{% macro create_entity_aggregate_dq_summary(entity_target_rule_binding_configs, gcp_project_id, gcp_bq_dataset_id, rule_binding_configs) -%}
 
 {% set rule_binding_ids_list = entity_target_rule_binding_configs.get('rule_binding_ids_list') %}
-
+{% set configs =  rule_binding_configs.get('configs') -%}
+{% set ns = namespace() %}
 {%- for rule_binding_id in rule_binding_ids_list -%}
-
+{% set ns.failed_records_sql_string = rule_binding_configs.get(rule_binding_id ~ '_failed_records_sql_string') -%}
```

Comment:
can we do this without namespace? e.g. pass `rule_binding_configs.get(rule_binding_id ~ '_failed_records_sql_string')` into line 79.